docs/src/main/asciidoc/grpc-kubernetes.adoc

```diff
@@ -0,0 +1,81 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Deploying your gRPC Service in Kubernetes
+include::_attributes.adoc[]
+:categories: serialization
+:summary: This guide explains how to deploy your gRPC services in Quarkus to Kubernetes.
+
+This page explains how to deploy your gRPC service in Quarkus in Kubernetes. 
+We'll continue with the example from xref:grpc-getting-started.adoc[the Getting Started gRPC guide]. 
+
+== Configuring your project to use the Quarkus Kubernetes extension
+
+Edit the `pom.xml` file to add the Quarkus Kubernetes extension dependency (just under `<dependencies>`):
+
+[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
+.pom.xml
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-kubernetes</artifactId>
+</dependency>
+----
+
+Next, we want to expose our application using the Kubernetes Ingress resource:
+
+[source,properties]
+----
+quarkus.kubernetes.ingress.expose=true
+----
+
+By default, the Quarkus Kubernetes will expose the HTTP port with name `http` which is bound to the Quarkus HTTP server, not the gRPC service. To expose the gRPC server instead, set the `quarkus.kubernetes.ingress.target-port=grpc` property in your application.properties:
+
+[source,properties]
+----
+quarkus.kubernetes.ingress.target-port=grpc
+----
+
+Finally, we need to generate the Kubernetes manifests by running the Maven command in a terminal:
+
+[source,shell]
+----
+$ mvn clean package
```

Comment:
May need the tabs with gradle and cli

Reply:
do you want me to update the rest of gRPC guides with this too ?

Reply:
If you can, that would be great.

I have a task somewhere about restructuring the whole gRPC doc. But didn't have the time to complete it.

Reply:
PR updated with this change.